tests/scripts/task_cpp_unittest.sh

```diff
@@ -28,38 +28,15 @@ else
     BUILD_DIR=build
 fi
 
-# Python is required by apps/bundle_deploy
-source tests/scripts/setup-pytest-env.sh
 
-export LD_LIBRARY_PATH="lib:${LD_LIBRARY_PATH:-}"
 # NOTE: important to use abspath, when VTA is enabled.
-export VTA_HW_PATH=`pwd`/3rdparty/vta-hw
+VTA_HW_PATH=$(pwd)/3rdparty/vta-hw
+export VTA_HW_PATH
 
 # to avoid CI thread throttling.
 export TVM_BIND_THREADS=0
 export OMP_NUM_THREADS=1
 
-# Build cpptest suite
-python3 tests/scripts/task_build.py \
-    --sccache-bucket tvm-sccache-prod \
-    --cmake-target cpptest \
-    --build-dir "${BUILD_DIR}"
-
-# crttest requries USE_MICRO to be enabled.
-if grep -Fq "USE_MICRO ON" ${BUILD_DIR}/TVMBuildOptions.txt; then
-  pushd "${BUILD_DIR}"
-  ninja crttest
-  popd
-fi
-
 pushd "${BUILD_DIR}"
```

Comment:
I'm not sure what happened here. shouldn't `ctest` also move to wherever `crttest` was built?

Reply:
The build stuff should all be moved into the corresponding build stage and downloaded in the relevant test step to actually be run, so this removes all the building and just runs ctest (the other stuff behind the flags is also moved to the other test script below)